CRM/Contribute/Form/Contribution.php

```diff
@@ -618,6 +618,29 @@ public function buildQuickForm() {
         TRUE, array('onChange' => "return showHideByValue('payment_instrument_id','4','checkNumber','table-row','select',false);")
       );
     }
+    
+    if ($this->_context == 'standalone') {
+      $this->assign('showRecurringField', 1);
+      $backOfficePaymentProcessors = CRM_Contribute_BAO_ContributionRecur::getBackOfficePaymentProcessors();
```

Comment:
Per previous comments - this call should be able to be replaced by a check for 

$this->_recurPaymentProcessors
(provided this line is after)
    $this->assignPaymentRelatedVariables();